README.md

```diff
@@ -50,6 +50,13 @@ SyntaxError: Use of const in strict mode.
 
 **Having installation troubles? Check out our [Troubleshooting guide](/TROUBLESHOOTING.md).**
 
+### Install from mirror in China
+
+```shell
+npx mirror-config-china --registry=http://registry.npm.taobao.org
```

Comment:
Can this be done without adding `npx` dependency? Something like  

    npm i -g mirror-config-china --registry=https://registry.npm.taobao.org

Reply:
Yes.